bin/mixer_codegen.sh

```diff
@@ -183,7 +183,7 @@ if [ "$opttemplate" = true ]; then
     die "template generation failure: $err";
   fi
 
-  go run "$GOPATH/src/istio.io/istio/mixer/tools/mixgen/main.go" api -t "$templateDS" --go_out "$templateHG" --proto_out "$templateHSP" "$TMPL_GEN_MAP"
+  go run $GOPATH/src/istio.io/istio/mixer/tools/mixgen/main.go api -t $templateDS --go_out $templateHG --proto_out $templateHSP $TMPL_GEN_MAP
```

Comment:
Sorry about [the ShellCheck job](https://circleci.com/gh/istio/istio/154099?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link) not properly reporting these (#7809 should fix it).

You need to keep the double quotes here.

Reply:
@Tahler : The problem is this that with quotes it's generating wrong filenames.. explained in detail in adapter one..

Reply:
`TMPL_GEN_MAP` should be an array. See the other comment.

Reply:
Thanks..Fixed